src/content/glossary/index.md

```diff
@@ -621,19 +750,27 @@ An off-chain scaling solution that uses [fraud proofs](#fraud-proof), like [Opti
   Plasma
 </DocLink>
 
+### presale {#presale}
+
+Sale of cryptocurrency before the actual launch of the network.
+
```

Comment:
Should we note that this was a form of initial ETH issuance that did not come from actual proof-of-work mining or proof-of-stake validating? Okay with just keeping it simple too, just a thought